packages/react-charting/src/components/PieChart/PieChart.base.tsx

```diff
@@ -56,10 +70,11 @@ export class PieChartBase extends React.Component<IPieChartProps, IPieChartState
           width={width!}
           height={height!}
           outerRadius={outerRadius}
-          innerRadius={0}
+          innerRadius={1}
```

Comment:
With inner Radius 0
![image](https://github.com/microsoft/fluentui/assets/70366079/fc557d99-a8e8-4ab3-9c07-a15e1815265f)

with inner Radius 1
![image](https://github.com/microsoft/fluentui/assets/70366079/7e7074cc-ddba-49fa-a9d8-207d281f2d89)

w/o using white colored stroke it's tricky (geometrically) to get a completely parellel spacing b/w the pies. We're using padAngle here to create the spacing b/w the pie, and since it's angular cut. a 1 px inner radius is required, to prevent it from intersecting in the center.

Reply:
We should add this comment in the code. And the images in the contributor guide.